.circleci/config.yml

```diff
@@ -432,6 +432,7 @@ commands:
     description: "Upload Coverage Report To codecov.io"
     steps:
       - codecov/upload:
+          flags: all
```

Comment:
Is `all` a special keyword, or we have it defined somewhere?

Is our eventual goal to split all into a collection of flags for separate modules?

Reply:
@derekpierre - This feature operates more like *tagging* than it does flags, and the input is arbitrary.  We can use these tags to aggregate several test reports into a single report on codecov.

> Is our eventual goal to split all into a collection of flags for separate modules?

 More aptly, we can tag individual *test suites* for coverage.